coaches_training.deck.html

```diff
@@ -89,19 +89,14 @@
 <h3>Who are you?</h3>
 
 <ul>
-<li>What's your name?</li>
-<li>What are your pronouns?</li>
-<li>Have you been to a ClojureBridge or similar workshops before? How many?</li>
-<li>What's your favorite place in Berlin?</li>
-<li>What's the most non-coding thing you do?</li>
+<li>What's your name and pronouns?</li>
+<li>Have you been to a ClojureBridge or similar workshops before?</li>
```

Comment:
I liked having one other non-programming related question. Maybe "fun fact about yourself" or "your favourite restaurant" or something. Or even "how did you hear about ClojureBridge"

Reply:
The reaosning behind this was that the whole time before you speak, you are worrying about what to say, and barely remember other people's answers anyway... At the same time, this part is also quite long.

Reply:
I am in favor of shortening it, but I hear Franka's point about keeping one (and only one) non-programming question. I vote "favorite place in Berlin" because it's easy and revealing without being intrusive.

Reply:
Given there are gonna be people not from Berlin there, I suspect another one would be better (fav. restaurant?) Though I am still personally skeptical about having it